test/tools/ossfuzz/CMakeLists.txt

```diff
@@ -9,41 +11,95 @@ add_dependencies(ossfuzz
         strictasm_assembly_ossfuzz
         )
 
+if (OSSFUZZ)
 add_custom_target(ossfuzz_proto)
 add_dependencies(ossfuzz_proto yul_proto_ossfuzz yul_proto_diff_ossfuzz)
+endif()
```

Comment:
This if case needs to be removed once we resolved how to locally compile proto-based fuzzer harnesses.